### ISSUE TITLE
gBar does not launch and takes up 100% CPU usage

### ISSUE THREAD

**Issue by Nizzi29** [scorpion-26/gBar#109]
**Describe the bug**
When trying to start gBar using `gBar bar 0` it does not open and the process is taking up 100% of the CPU

**Steps to Reproduce**
1. Follow steps under "Building and installation (Manually)"
2. `/usr/local/bin/gBar bar 0`

(Same problem with `yay -S gbar-git`)

**Expected behavior**
gBar should be visible and not take up 100% CPU

**Screenshots/Error logs**
![image](https://github.com/user-attachments/assets/97bb5b0b-5771-408d-944a-1dbce5aaa80c)

[config.txt](https://github.com/user-attachments/files/17356997/config.txt)

 
[gBar-9866.log](https://github.com/user-attachments/files/17356990/gBar-9866.log)

**Information about your system and gBar**
 - OS: Arch
 - Hyprland

**Comment by scorpion-26** [comment]
Do you have a pulseaudio server/daemon running (pipewire-pulse counts too)? If not, then that is likely the issue.

**Comment by Nizzi29** [comment]
So I've got it to work now and was about to delete this post since it wasn't needed anymore but since you already answered and I guess it could help someone in the future:

I was using pulseaudio and checked with `htop` to make sure it was running so I didn't think that was the problem. Regardless I changed my hyprland.conf so that it would start pipewire instead of pulseaudio when launching because I read somewhere that it is somewhat more compatible with a lot of stuff than pulseaudio. I rebootet and startet bluetooth.service manually before starting Hyprland because I don't know how to automate that yet. Thats all I changed. So I guess switching to pipewire solved it?

I'm still really new to the whole Arch / "configuring your entire operating system manually" stuff. I'm glad it's working now so I can finally move on to the next problem thats gonna keep me from ricing my system :D

Thanks a lot for the quick answer, looking forward to using gBar.
Love your "Implementing it myself is fun and a great excuse to learn something new!" approach btw!
